perfdash/config.go

```diff
@@ -131,16 +145,30 @@ func getProwConfig() (Jobs, error) {
 	}
 	jobs := Jobs{}
 	for _, periodic := range conf.Periodics {
+		var thisPeriodicConfig Tests
 		for _, tag := range periodic.Tags {
 			if strings.HasPrefix(tag, "perfDashPrefix:") {
 				split := strings.SplitN(tag, ":", 2)
-				jobs[periodic.Name] = Tests{
-					Prefix:       strings.TrimSpace(split[1]),
-					Descriptions: performanceDescriptions,
-				}
-				break
+				thisPeriodicConfig.Prefix = strings.TrimSpace(split[1])
```

Comment:
Check before this that split actually is of length at least 2?

Reply:
I do, 2nd arg to HasPrefix contains a colon.

Reply:
Ah that's true.